Update Licensing topic for 24.1 (#6350)
DevExpress/devextreme-documentation: concepts/Common/Licensing/25 Validation.md

[note] DevExtreme’s license verification process does not impact app functionality nor affect app performance. 

> When creating the first component, DevExtreme checks the license key specified in the `config()` method. If license key verification is successful, you will not see any additional license-related messages within the browser console or on-screen.

Reviewer: ```suggestion
When creating the first component, DevExtreme checks the license key specified in the `config()` method. If license key verification is successful, additional license-related messages are not shown within the browser console or on-screen.
```
